src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -356,10 +332,9 @@ private void AddCoins(TransactionBuildContext context)
         /// <remarks>
         /// Add outputs to the <see cref="TransactionBuilder"/> based on the <see cref="Recipient"/> list.
         /// </remarks>
-        private void AddRecipients(TransactionBuildContext context)
+        public virtual void AddRecipients(TransactionBuildContext context)
```

Comment:
protected

Reply:
Ah this needs to be virtual as we override the behavior in SmartContractWalletTransactionHandler